index.html

```diff
@@ -1,7 +1,7 @@
 <html>
-  <head> </head>
+  <head> <Link href="style.css" rel="stylesheet" type="text/css"></head>
   <body>
     <h1 class="header">Hola Mundo</h1>
-    <h2>Que onda</h2>
+    <h2 class="header_2">Que onda</h2>
```

Comment:
There is a conventional naming for classes and ids on html which is using "-" instead "_"